### ISSUE TITLE
Fix aria-expanded on nodes to only exist when a node can expand/collapse

### ISSUE THREAD

**Issue by grapoza** [grapoza/vue-tree#209]
The current behavior of `aria-expanded` on static tree nodes (those that cannot expand or collapse) is to exist but be set to `false` instead of the proper behavior, which is to not exist on that node at all. W3C has [an example](https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions#Example_3:_Omitting_aria-expanded_produces_a_static_tree) of the correct usage.
